SU2_CFD/src/variables/CVariable.cpp

```diff
@@ -77,8 +78,14 @@ CVariable::CVariable(unsigned long npoint, unsigned long ndim, unsigned long nva
 	if (config->GetFSI_Simulation() && config->GetDiscrete_Adjoint()) {
 	  Solution_Adj_Old.resize(nPoint,nVar);
 	}
-
+  
+  /* Under-relaxation parameter. */
+  UnderRelaxation.resize(nPoint) = 1.0;
+  LocalCFL.resize(nPoint) = 0.0;
+  
+  /* Non-physical point (first-order) initialization. */
   Non_Physical.resize(nPoint) = false;
+  Non_Physical_Counter.resize(nPoint) = 0;
```

Comment:
Can you allocate these only where needed? Some of the classes inheriting from CVariable that use this constructor do not need these fields (DiscAdj variables, FEA variables). Maybe we need more overloads...

Reply:
Done